content/post/language_migrations.md

```diff
@@ -0,0 +1,512 @@
+      <table>
+    <tr>
+      <th>Rank</th>
+      <th>Language</th>
+      <th>Popularity, %</th>
+      <th>Source code, %</th>
+    </tr>
+    </table>
+    <div class="inner_table">
+    <table>
+    <tr>
+      <td>1.</td>
+      <td>Python</td>
+      <td>16.1</td>
+      <td>11.3</td>
+    </tr>
+    <tr>
+      <td>2.</td>
+      <td>Java</td>
+      <td>15.3</td>
+      <td>16.6</td>
+    </tr>
+  <tr>
+    <td>3.</td>
+    <td>C</td>
+    <td>9.2</td>
+    <td>17.2</td>
+  </tr>
+  <tr>
+    <td>4.</td>
+    <td>C++</td>
+    <td>9.1</td>
+    <td>12.6</td>
+  </tr>
+  <tr>
+    <td>5.</td>
+    <td>PHP</td>
+    <td>8.5</td>
+    <td>24.4</td>
+  </tr>
+  <tr>
+    <td>6.</td>
+    <td>Ruby</td>
+    <td>8.3</td>
+    <td>2.6</td>
+  </tr>
+  <tr>
+    <td>7.</td>
+    <td>C#</td>
+    <td>6.1</td>
+    <td>6.5</td>
+  </tr>
+  <tr>
+    <td>8.</td>
+    <td>Objective-C</td>
+    <td>4.0</td>
+    <td>3.3</td>
+  </tr>
+  <tr>
+    <td>9.</td>
+    <td>Go</td>
+    <td>3.2</td>
+    <td>0.7</td>
+  </tr>
+  <tr>
+    <td>10.</td>
+    <td>Swift</td>
+    <td>2.6</td>
+    <td>0.3</td>
+  </tr>
+  <tr>
+    <td>11.</td>
+    <td>Scala</td>
+    <td>2.2</td>
+    <td>0.3</td>
+  </tr>
+  <tr>
+    <td>12.</td>
+    <td>Perl</td>
+    <td>2.0</td>
+    <td>0.9</td>
+  </tr>
+  <tr>
+    <td>13.</td>
+    <td>R</td>
+    <td>1.8</td>
+    <td>0.3</td>
+  </tr>
+  <tr>
+    <td>14.</td>
+    <td>Haskell</td>
+    <td>1.8</td>
+    <td>0.2</td>
+  </tr>
+  <tr>
+    <td>15.</td>
+    <td>Lua</td>
+    <td>1.7</td>
+    <td>0.7</td>
+  </tr>
+  <tr>
+    <td>16.</td>
+    <td>Matlab</td>
+    <td>1.7</td>
+    <td>0.5</td>
+  </tr>
+  <tr>
+    <td>17.</td>
+    <td>Clojure</td>
+    <td>1.5</td>
+    <td>0.2</td>
+  </tr>
+  <tr>
+    <td>18.</td>
+    <td>Rust</td>
+    <td>1.2</td>
+    <td>0.1</td>
+  </tr>
+  <tr>
+    <td>19.</td>
+    <td>Erlang</td>
+    <td>1.0</td>
+    <td>0.1</td>
+  </tr>
+  <tr>
+    <td>20.</td>
+    <td>Visual Basic</td>
+    <td>1.0</td>
+    <td>0.3</td>
+  </tr>
+  <tr>
+    <td>21.</td>
+    <td>Fortran</td>
+    <td>0.9</td>
+    <td>0.3</td>
+  </tr>
+  <tr>
+    <td>22.</td>
+    <td>Pascal</td>
+    <td>0.8</td>
+    <td>0.6</td>
+  </tr>
+</table>
+</div>
+</div>
+
+<p align="center" class="caption">Popularity of languages according to
+<a href="https://en.wikipedia.org/wiki/Centrality">centrality measure</a> on GitHub</p>
+
+**Python** (16.1 %) appears to be most attractive language, followed closely by **Java** (15.3 %). However, it's all the more significant because only 11.3 % of all source codes written on GitHub, in one of those 22 languages, have been in **Python**.
+
+In Erik's ranking, **Go** was the big winner with 16.4 %. Since Erik based his approach on Google queries, it seems that the buzz around Go that makes people wondering explicitly on blogs if they should move to this language, takes a bit of time to become apparent in terms of projects effectively written in **Go** on GitHub.
```

Comment:
makes people wonder https://www.youtube.com/watch?v=sAebYQgy4n4

~on~ in blogs